src/server/api/routers/bot.ts

```diff
@@ -187,6 +192,30 @@ export const botRouter = createTRPCRouter({
       return bot
     }),
 
+  getBotAppearanceSettingByApiToken: integrationProcedure.query(
+    async ({ ctx }) => {
+      const sources = await db.query.bots.findFirst({
```

Comment:
Should not be plural.

Should select necessary fields. Shouldn't fetch all bot field.